src/api/config/config.go

```diff
@@ -37,6 +37,7 @@ type Configuration struct {
 	OAuthConfig          oauthConfig      `yaml:"oauthConfig"`
 	SigningKey           string           `yaml:"signingKey"`
 	Initialized          bool
+	TillerNamespace      string `yaml:"tillerNamespace"`
```

Comment:
Have you run [gofmt](https://golang.org/cmd/gofmt/) on your code? It should correctly align this

Reply:
this was fixed

Reply:
@karmab it still doesn't look properly aligned?

Reply:
actually you were right but we're good now!